R/prepare_ictr_product.R

```diff
@@ -3,12 +3,7 @@
 #' @description
 #' `r lifecycle::badge("deprecated")`
 #'
-#' @param match_mapper A dataframe like [matches_mapper]
-#' @param eco_activities A dataframe like [ecoinvent_activities]
-#' @param ictr_prod A dataframe like [ictr_product]
-#' @param comp A dataframe like [ep_companies]
-#' @param eco_inputs A dataframe like [ecoinvent_inputs]
-#' @param isic_tilt_map A dataframe like [isic_tilt_mapper]
+#' @inheritParams prepare_ictr_company
```

Comment:
This and similar changes are not strictly necessary but remove lots of instances of "isic_tilt_map" which make it easier to use refactoring tools to detect instances of "isic_tilt" that needed to change to "isic".

Also it brings us a step closer to retiring the old api by no longer exporting it -- which is something we may be able to do soon without disrupting others.